### ISSUE TITLE
fine-tuning 중에 Killed 라는 메세지와 함께 종료가 됩니다. 

### ISSUE THREAD

**Issue by ethany21** [yoonhero/jamo_llm#4]
현재 windows 10의 wsl 환경에서 jamo 프로젝트 파인 튜닝을 하는 중입니다.

`python3 finetune/full.py --model_path "model/jamo_2b_finetuned.tar" --model_size "small" --learning_rate 0.00003 --batch_size 1 --gradient_accumulate 4 --max_iters 100 --warmup_iters 20 --save_interval 20 --cache_path "data/sft-cache.hdf5" --result_checkpoint_dir "model/fine_tuned" --with_lr_scheduler`

데이터셋의 개수는 55개이고, CPU를 활용해서 full fine tuning을 하고자 했습니다.

windows 터미널 환경에서 wsl 실행하고 finetune/full.py 스크립트를 위 명령어로 실행했는데

"Finishing Loading the DataLoader",
"Iter {iteration}: Eval Loss = {min_loss}"

로그 메세지 없이 프로그램이 작동하다 
Killed라는 메세지가 뜨고 도중에 중단이 되었습니다.

스크립트 실행 시 paratemter들 (max_iters , warmup_iters , save_interval ) 등을 바꿔 보며 실행 중인데
결과는 동일한 현상이 있습니다.

혹시 관련해서 확인 가능할까요?

**Comment by yoonhero** [comment]
혹시 requirements.txt에 있는 패키지를 다 설치하고 실행했는데, 이런 문제가 발생하시나요?
main directory에 있는 training.log 파일을 캡처해서 보여주실 수 있나요?

**Comment by ethany21** [comment]
requirements.txt에 있는 패키지는 전부 설치했습니다.

pycharm ide에서 실행한 것이 아니라 wsl에서 cli로 실행하고 있는데 main directory(README.md, requirements.txt 가 있는 directory)에 training.log 파일이 따로 생성되지는 않았습니다.

`python3 finetune/full.py --model_path "model/jamo_2b_finetuned.tar" --model_size "small" --learning_rate 0.00003 --batch_size 1 --gradient_accumulate 4 --max_iters 100 --warmup_iters 20 --save_interval 20 --cache_path "data/sft-cache.hdf5" --result_checkpoint_dir "model/fine_tuned" --with_lr_scheduler >> training.log 2>&1`

상단의 명령어 끝에 >> training.log 2>&1 명령어로 training.log 파일에 stdout으로 출력하고 결과 확인해 보면 될까요?

**Comment by yoonhero** [comment]
training.log가 생성되지 않았다는 것을 보았을 때, 아마도 cli로 입력하는 경로에서 문제가 생긴 것 같습니다. 혹시 model/jamo_2b_finetuned.tar과 같은 경로를 ./model/jamo_2b_finetuned.tar과 같이 바꾸어 보시고, 경로가 맞게 입력 되었는지와 output directory를 생성하고 재시도해도 오류가 발생하면 재문의 부탁드립니다.
